recipes/hklpy2/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "hklpy2" %}
+{% set version = "0.0.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hklpy2-{{ version }}.tar.gz
+  sha256: 159033cc0b81f9ac6f803ce50d39363cb704f64bb99d829630d13eaf42a30efc
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [python_impl == 'pypy']
+  skip: true  # [py<310 or win or osx]
+
```

This looks like it can be noarch. I did not see any reason to skip Windows or mac OS.
If the reason is a missing dependency on those platforms, we still prefer noarch to safe resources. The package won't be installable there and would be ready if/when the missing dependencies are available.

```suggestion
  noarch: python

```

---

Beginning to understand this more.

I should add the `hkl` package but mark it as linux 64-bit only..
Also, the associated entry-point should also be linux 64-bit only.
I need to learn how to do that.

---

And the unit tests involving the `hkl` package need to be made Linux 86-bit only.

---

Linux-only for now.

---

It is OK for noarch package to fails on platforms with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on those platforms. TL;DR, if the reason is only b/c `hkl` is missing on other platforms, it should still be noarch.

---

> OK for noarch package to fails on platforms with a missing dependency

Now, I understand.  Switched to `noarch: python` and added version minimum for Python.  Builds as expected.